src/ui/mathquill-editor/MathQuillEditorSymbols.js

```diff
@@ -7,229 +7,467 @@ export type MathQuillEditorSymbol = {
+  cmd: 'write',
+};
+export const PI = {
+  label: '\\pi',
+  latex: '\\pi',
+  description: 'Pi',
+  cmd: 'write',
+};
+export const PLUS = {
+  label: '+',
+  latex: '+',
+  description: 'Addition',
+  cmd: 'cmd',
+};
+export const PM = {
   label: '\\pm',
   latex: '\\pm',
   description: 'Plus-Minus',
   cmd: 'cmd',
 };
-const SIN = {label: '\\sin', latex: '\\sin', description: 'Sin', cmd: 'write'};
-const SQRT2 = {
+export const POWER = {
+  label: 'x^{y}',
+  latex: '^',
+  description: 'Power',
+  cmd: 'cmd',
+};
+export const SIMEQ = {
+  label: '\\simeq',
+  latex: '\\simeq',
+  description: 'Approximately Equal',
+  cmd: 'write',
+};
+export const SIM = {
+  label: '\\sim',
+  latex: '\\sim',
+  description: 'Similarity',
+  cmd: 'write',
+};
+export const SIN = {
+  label: '\\sin',
+  latex: '\\sin',
+  description: 'Sin',
+  cmd: 'write',
+};
+export const SMALLE = {
+  label: '\u212F',
+  latex: '\u212F',
+  description: 'Script Small E',
+  cmd: 'write',
+};
+export const SQR = {
+  label: 'x^{2}',
+  latex: '^{2}',
+  description: 'Square',
+  cmd: 'write',
+};
+export const SQRT2 = {
   label: '\\sqrt[x]{y}',
   latex: '\\sqrt[x]{y}',
   description: 'Square Root Alt',
   cmd: 'write',
 };
-const SQRT = {
+export const SQRT = {
   label: '\\sqrt x',
   latex: '\\sqrt',
   description: 'Square Root',
   cmd: 'cmd',
 };
-const SUBSCRIPT = {
+export const SUBSCRIPT = {
   label: 'x_{2}',
   latex: '_{2}',
   description: 'Subscript',
   cmd: 'write',
 };
-const SUM = {
+export const SUBSET = {
+  label: '\\sub',
+  latex: '\\sub',
+  description: 'Subset',
+  cmd: 'write',
+};
+export const SUBSETEQ = {
+  label: '\\sube',
+  latex: '\\sube',
+  description: 'Subset or Equal',
+  cmd: 'write',
+};
+export const SUM = {
   label: '\\sum',
   latex: '\\sum',
   description: 'Summation',
   cmd: 'cmd',
 };
-const SUPERSCRIPT = {
-  label: 'x^{super}',
-  latex: '^{super}',
-  description: 'Exponent',
+export const SUPERSET = {
+  label: '\\supset',
+  latex: '\\supset',
+  description: 'Superset',
+  cmd: 'write',
+};
+export const SUPERSETEQ = {
+  label: '\\supe',
+  latex: '\\supe',
+  description: 'Superset or Equal',
   cmd: 'write',
 };
-const TAN = {label: '\\tan', latex: '\\tan', description: 'Tan', cmd: 'write'};
-const THETA = {
+export const TAN = {
+  label: '\\tan',
+  latex: '\\tan',
+  description: 'Tan',
+  cmd: 'write',
+};
+export const THETA = {
   label: '\\theta',
   latex: '\\theta',
   description: 'Theta',
   cmd: 'write',
 };
-const TIMES = {
+export const TIMES = {
   label: '\\times',
   latex: '\\times',
   description: 'Multiplication',
   cmd: 'cmd',
 };
-const TRIANGLE = {
+export const TRIANGLE = {
   label: '\\bigtriangleup',
   latex: '\\bigtriangleup',
   description: 'Triangle',
   cmd: 'write',
 };
-const X = {label: 'x', latex: 'x', description: 'x', cmd: 'write'};
-const Y = {label: 'y', latex: 'y', description: 'y', cmd: 'write'};
-
+export const UNDERLINE = {
+  label: '\\underline\u3000',
```

Comment:
`label` intentionally differs from the `latex` in this one?

Reply:
Yes.

`\u3000'` shows the wide space character and the label will look like "___".
`\\underline` is just the LateX style and "\u3000" is the text to be styled (with underline).